docs/source/search-syntax.rst

```diff
@@ -148,6 +148,13 @@ search programmatically. You can specify the list of columns to order by
 optional ``DESC`` or ``ASC`` value; the default is ``ASC``. The default ordering is to sort by
 ``start_time DESC``, then ``run_id``.
 
+The :py:func:`mlflow.search_runs` can be used to search for runs within specific experiments which
+can be identified by experiment IDs or experiment names, but not both at the same time.
+
```

Comment:
```suggestion
The :py:func:`mlflow.search_runs` API can be used to search for runs within specific experiments. Experiments
can be identified by ID or by name, but not both at the same time.

```

Reply:
Is there a way to accept to accept the suggested change and sing it via GitHub UI? Last time I've accepted one I ended up with an unsigned commit so the build failed